apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -13,23 +13,24 @@ import PaneHeader, { PaneButton } from '../../templates/PaneHeader';
 import experiments from '@cdo/apps/util/experiments';
 import InstructionsTab from './InstructionsTab';
 import HelpTabContents from './HelpTabContents';
+import {toggleInstructionsCollapsed,
+   setInstructionsMaxHeightNeeded,
+   setInstructionsRenderedHeight,
+   setInstructionsHeight} from '../../redux/instructions';
```

Comment:
Not sure how to enforce this with our linter, but I think we typically format these multiline imports like this:
```js
import {
  toggleInstructionsCollapsed,
  setInstructionsMaxHeightNeeded,
  setInstructionsRenderedHeight,
  setInstructionsHeight,
} from '../../redux/instructions';
```